src/components/Category.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div class="feedback-container__category">
```

Comment:
Good idea to make each category its own component! This will def help DRY things up.